doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -786,3 +787,351 @@ JCL      - fd5cc89 based on jdk-11.0.2+7)
+```
+poweroff
+```
+
+[6] Mount the Fedora OS image to your system for the cross-compilation.
+```
+mkdir <your_fedora_mount_directory>
+sudo mount -o loop  stage4-disk.img  <your_fedora_mount_directory>
+
+<your_fedora_mount_directory># ls
+bin   dev  home  lib64       media  opt   root  sbin  sys  usr
+boot  etc  lib   lost+found  mnt    proc  run   srv   tmp  var
+```
+
+### 3. Compile the cross-toolchain for RISC-V
+:rocket:
+The GNU cross-toolchain project contains everything required during compilation except the OS specific headers/libraries which are already installed previously on the Fedora OS image. 
+
+[1] Run the following command to get the source of the cross-toolchain:
+```
+$ git clone --recursive https://github.com/riscv/riscv-gnu-toolchain
+```
+
+[2] When the process is complete, install all required software packages on your host system:
+```
+$ sudo apt-get install autoconf \
+automake \
+autotools-dev \
+curl \
+libmpc-dev \
+libmpfr-dev \
+libgmp-dev \
+gawk \
+build-essential \
+bison \
+flex \
+texinfo \
+gperf \
+libtool \
+patchutils \
+bc \
+zlib1g-dev \
+libexpat-dev
+```
+
+[3] Set up the installation path and build the cross-toolchain on your system:
+```
+cd riscv-gnu-toolchain
+./configure --prefix=/opt/riscv_toochain_linux (the path to install the cross-toolchain)
+make linux
+```
+
+:bulb:
+Please refer to the latest instructions at `https://github.com/riscv/riscv-gnu-toolchain` for details if
+you are working on other host systems or if any issue is detected when compiling the cross-toolchain.
+
+[4] Run the RISC-V specific gcc command to ensure the cross-toolchain works good as expected:
+```
+ $ riscv64-unknown-linux-gnu-gcc --version
+   riscv64-unknown-linux-gnu-gcc (GCC) 9.2.0
+   Copyright (C) 2019 Free Software Foundation, Inc.
+   This is free software; see the source for copying conditions.
+   There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+```
+
+### 4. Get the source for building the JDK
+:rocket: You need to clone the Extensions for OpenJDK for OpenJ9 project, which is a git mirror of OpenJDK without the HotSpot JVM, but with an **openj9** branch that contains a few necessary patches. 
+
+[1] Run the following command to close the Extensions for OpenJDK:
+```
+git clone https://github.com/ibmruntimes/openj9-openjdk-jdk11.git
+```
+
+[2] Change directory into the cloned repository when the process is complete:
+```
+cd openj9-openjdk-jdk11
+```
+:bulb:
+Cloning this repository can take a while due to the huge size of the OpenJDK project
+
+[3] Now fetch additional sources from the Eclipse OpenJ9 project and its clone of Eclipse OMR:
+```
+bash get_source.sh
+```
+
+### 5. Configure the cross-compiling environment
+:rocket:
+When you have all the source files on the host system, run the following configure command to set up the cross-compilation environment:
+```
+bash configure --disable-warnings-as-errors \ 
+               --enable-ddr=no \
```

Comment:
This would normally be expressed as `--disable-ddr`.

Reply:
Agreed and replaced with `--disable-ddr`.